### ISSUE TITLE
Binary Classifier experiment - log loss

### ISSUE THREAD

**Issue by andrewcz** [twitter/torch-autograd#97]
Hi,
Fantastic Library!
I was just wondering, i am trying to use the library for a binary classifier experiment using the log loss function to train the model. This is for a university experiment around benchmarking different models. Would you have time to provide an example of how to use the library to achieve the above goal.
Also, with an included visualisation on how the model learns.
Many thanks,
Best,
Andrew

**Comment by alexbw** [comment]
Hey Andrew, I think you might find that open source library maintainers are a bit bandwidth constrained to cook up custom examples. For example:

https://github.com/fchollet/keras/issues/2007
https://github.com/automl/auto-sklearn/issues/61
https://github.com/uci-cbcl/DanQ/issues/2
https://github.com/HIPS/autograd/issues/92
https://github.com/dmlc/minerva/issues/52
https://github.com/beniz/deepdetect/issues/81
https://github.com/NervanaSystems/neon/issues/212

However, you might consult the excellent books "Pattern Recognition and Machine Learning" by Bishop and "Machine Learning: A Probabilistic Perspective" by Murphy, and of course, keep monitoring the other threads you've opened. Best of luck.
